app/models/blueprint.rb

```diff
@@ -8,6 +10,9 @@ class Blueprint < ApplicationRecord
 
   acts_as_miq_taggable
 
+  after_create :set_original_blueprint
+  default_value_for :active_version, false
```

Comment:
This attribute works around having to do special db queries to determine the "latest" version.
This seems like a nice approach.

But I had expected this to default `active_version` to true. and setting all other previous versions to false on the `after_save`.